client/src/components/Auth/AuthProvider.js

```diff
@@ -32,7 +32,7 @@ const AuthProvider = ({ children }) => {
                 { email: email, password: password }, 
                 { 'content-type': 'application/json' }
             )
-            const something = registerResponse.data.token
+            const something = jwt(registerResponse.data.token)
```

Comment:
AFAIK this function doesn't actually decode the token. https://github.com/auth0/jwt-decode